docs/usage.md

```diff
@@ -62,67 +62,101 @@ curl -X POST \
 
 ## Kafka topics
 
-Maestro can be configured as mentioned under the running configurations section to listen to Kafka topics
+Maestro can be configured to listen to Kafka topics as described in the Running Configurations section.
 
 ```yaml
-MAESTRO_KAFKA_ENABLED=true
-MAESTRO_KAFKA_SERVERS=http://kafka:9092
-MAESTRO_KAFKA_INDEX_REQUEST_TOPIC=maestro_index_request
-MAESTRO_KAFKA_INDEX_REQUEST_DLQ=maestro_index_request_dlq
-MAESTRO_KAFKA_LYRIC_ANALYSIS_MESSAGE_TOPIC=clinical_data
-MAESTRO_KAFKA_LYRIC_ANALYSIS_MESSAGE_DLQ=clinical_data_dlq
-MAESTRO_KAFKA_SONG_ANALYSIS_MESSAGE_TOPIC=song_analysis
-MAESTRO_KAFKA_SONG_ANALYSIS_MESSAGE_DLQ=song_analysis_dlq
+MAESTRO_KAFKA_SERVER=http://kafka:9092
 ```
 
-`MAESTRO_KAFKA_ENABLED` must be set to true in order to enable Kafka functionality.
+The `MAESTRO_KAFKA_SERVER` configuration specifies the Kafka server(s) to connect to.
```

Comment:
this gives the impression you can connect to multiple separate Kafka instances, is that so?
in that case, how many, and how do you specify different urls? (I don't see the usual _1, _2, etc pattern)